CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+#### [v0.9.0](https://github.com/heptio/ark/releases/tag/v0.9.0) - 2018-07-05
+
+##### Highlights:
+  * Ark now has support for backing up and restoring Kubernetes volumes using a free open-source backup tool called [restic](https://github.com/restic/restic).
+    This provides users an out-of-the-box solution for backing up and restoring almost any type of Kubernetes volume, whether or not it has snapshot support
+    integrated with Ark. For more information, see the [documentation](https://github.com/heptio/ark/blob/master/docs/restic.md).
+  * Support for Prometheus metrics has been added! Get more visibility into the behavior of Ark.
```

Comment:
Maybe something like: "View total number of backup attempts, including success or failure, total back up size in bytes, and backup durations with more metrics to come"

Wording could probably use work, but those are the metrics that'll be making the cut.